docs/DEVELOPER-ADVANCED.md

```diff
@@ -24,7 +24,9 @@ To get your Ubuntu 16.04 or 18.04 LTS install up and running to develop Discours
 
     rvm install 2.6.2
     rvm --default use 2.6.2 # If this error out check https://rvm.io/integration/gnome-terminal
-    gem install bundler mailcatcher rake
```

Comment:
I think we still need to install `bundler` and `rake` here?

Reply:
Good catch! I've made the required change.